### PR TITLE
refactor: migrate tx requests tests to playwright, closes #4151

### LINKED ISSUE #4151: Migrate transactions test to playwright
High prio

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Leather — download [extension builds](https://github.com/leather-wallet/extension/actions/runs/6087701431).

Migrates the transaction request tests to playwright.

Also, moved some selectors here out of `TestAppSelectors`. cc @alter-eggo, what do you think? I feel like only selectors being used in the test-app should be located there? Now, `UpdateProfileRequestSelectors`?

### CODE REVIEW

File: tests/selectors/test-app.selectors.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 export enum TestAppSelectors {
+  BtnContractCall = 'btn-contract-call',
   BtnUpdateValidProfile = 'btn-update-valid-profile',
   BtnUpdateInvalidProfile = 'btn-update-invalid-profile',
```
Comment (alter-eggo) [inline tests/selectors/test-app.selectors.ts:5]: hm, so may be its better to move `BtnUpdateValidProfile` and `BtnUpdateInvalidProfile` to `UpdateProfileRequestSelectors` ?

Comment (fbwoolf) [reply tests/selectors/test-app.selectors.ts:5]: Those actually live in the test-app code, so I was going to leave those.

**Review by alter-eggo — APPROVED** [review]

File: tests/page-object-models/transaction-request.page.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  async getDisplayedFeeValue() {
+    return this.page.locator(this.feeToBePaidLabel).innerText();
+  }
+
+  async waitForTransactionRequestError(msg: string) {
+    return this.page.waitForSelector(`text=${msg}`);
+  }
+
+  // async getTransactionRequestError() {
```
Comment (alter-eggo) [inline tests/page-object-models/transaction-request.page.ts]: should be removed?

**Review by pete-watters — COMMENTED** [review]
Nice work 👍

**Pull request merged**